examples/emission_events.py

```diff
@@ -136,16 +147,19 @@ def get_emission_events(
     # note, fluor.extCoeff is already a pint Quantity of units 1/M/cm
     ext_coeff = ex_spectrum[:, 1] * fluor.extCoeff
 
-    # calculate the number of photons hitting a fluorophore per second
+    # calculate the number of photons getting aborbed by a fluorophore per second
     exc_rate = fluorophore_photon_flux(wavelengths, irradiance, ext_coeff)
 
     # if the fluorophore has a lifetime, calculate the effective excitation rate
     # based on the fraction of time in the excited state
     # (i.e. at some point, we'll hit ground state depletion and saturation effects)
     if (lifetime := getattr(fluor, "lifetime", None)) is not None:
         lifetime = _ensure_quantity(lifetime, "ns")
+        # NOTE: assuming 1 fluorophore absorbs 1 photon, exc_rate is also the number of
+        # fluorophores which are in excited state per second.
         # Calculate the fraction of time in the excited state
         f_excited = exc_rate * lifetime
```

Comment:
When. I see the rate [equation](https://chem.libretexts.org/Courses/Providence_College/CHM_331_Advanced_Analytical_Chemistry_1/10:_Molecular_Luminescence_Spectrometry/10.03:_Applications_of_Photoluminescence_Methods/10.3.04:_The_Fluorescence_Lifetime_and_Quenching), I don't understand this equation.